fabric-provider-api-v1/src/main/java/net/fabricmc/fabric/api/provider/v1/ApiProviderAccess.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.provider.v1;
+
+/**
+ * Provides loosely-coupled access to providers of API associated with game objects.
+ *
+ * @param <P> Identifies the API provider type
+ * @param <A> Identifies the API type
+ */
+public interface ApiProviderAccess<P extends ApiProvider<P, A>, A> {
+	/**
+	 * API instance to be returned when the API is not present or not available.
+	 *
+	 * @return instance to be returned when the API is not present or not available.
+	 */
+	A absentApi();
```

Comment:
Do we want to call this `getAbsentApi`? `getBlah` allows some languages like kotlin to infer the method as a `val` and makes access in those languages much nicer

Reply:
Yeah, especially since Yarn uses bean prefixes for accessor methods.